CRM/Upgrade/Incremental/sql/5.15.alpha1.mysql.tpl

```diff
@@ -1 +1,11 @@
 {* file to handle db changes in 5.15.alpha1 during upgrade *}
+--dev/core#905 Add contribution recur status option group
+INSERT INTO `civicrm_option_group`  ( `name`, {localize field='title'}`title`{/localize}, `is_active` ) VALUES ('contribution_recur_status', {localize}'{ts escape="sql"}Contribution Recur Status{/ts}'{/localize}, 1);
```

Comment:
Contribution status option group has is_reserved and is_locked to TRUE, should this option should have same value for those fields?

Reply:
@pradpnayak yes, thank you - I will fix!